os/drivers/lcd/lcd_dev.c

```diff
@@ -319,6 +333,34 @@ static int lcddev_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 	return ret;
 }
 
+int lcd_flushing_thread(int argc, char *argv[])
```

Comment:
This is a new feature in the LCD driver and it is not mandatory, so it seems better to enable it through new config.